index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
![](https://c.tenor.com/0Q8SiDHvSZIAAAAC/so-close-so-close-seth-meyers.giff)

The work is looking perfect so far.
I think there are a few key configurations you are still missing.
In other for this project to follow the proper git flow, I think you need to set up a few files.

-  add **.gitignore** to your project
-  **Linters**  [ link to the linters life, find setup instructions](https://github.com/microverseinc/linters-config)